src/main/java/com/ashkan/moviesapi/controllers/ActorController.java

```diff
@@ -0,0 +1,35 @@
+package com.ashkan.moviesapi.controllers;
+
+import com.ashkan.moviesapi.entities.Actor;
+import com.ashkan.moviesapi.services.interfaces.ActorService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/actors")
+public class ActorController {
+    private ActorService actorService;
+
+    @Autowired
+    public ActorController(ActorService actorService) {
+        this.actorService = actorService;
+    }
+
+    @GetMapping
+    public List<Actor> findAll() {
+        return actorService.findAll();
+    }
+
+    @GetMapping("{Id}")
```

Comment:
Por favor confirmar si el path del endpoint está bien construido, no estas addicionando **/** antes de {Id}, el path final debería ser: _GET ... /api/actors/Id_

En una nota aparte:
No estas usando la propiedad **produces = MediaType.APPLICATION_JSON_VALUE** en la definición de tu endpoint, asumo que esto está funcionando bien y retorna un objeto json por defecto,  en proyectos donde necesitoes utilizar algún framework para generar documentación como Swagger por ejemplo necesitarás especificar el tipo de MediaType en la definición  de tu endpoint para que Swagger pueda generar la documentación correctamente.

Reply:
Path is builded correctly .
By default a JSON is returned ,but i will be researching about MediaType